metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/service/Converter.java

```diff
@@ -233,9 +238,33 @@ static EnrichmentTerm organizationImplToEnrichmentTerm(
     enrichmentTerm.setCreated(Objects.requireNonNullElseGet(created, Date::new));
     enrichmentTerm.setUpdated(updated);
 
+    //generate LabelInfos
+    final Map<String, List<String>> combinedLabels = new HashMap<>();
+    copyToCombinedLabels(combinedLabels, organizationEnrichmentEntity.getPrefLabel());
+    copyToCombinedLabels(combinedLabels, organizationEnrichmentEntity.getAltLabel());
+    copyToCombinedLabels(combinedLabels, organizationEnrichmentEntity.getEdmAcronym());
```

Comment:
These two blocks seem to be a copy from the tool that we have to populate enrichment but with the addition of EdmAcronym. Is this field a particularity only for organization entities?